### ISSUE TITLE
DET:Problem with file - <library count="0">

### ISSUE THREAD

**Issue by Omertron** [YAMJ/yamj-v2#2340]
Original [issue 2341](https://code.google.com/p/moviejukebox/issues/detail?id=2341) created by Omertron on 2012-02-13T15:17:42.000Z:

<b>What steps will reproduce the problem?</b>
<b>1.</b>
<b>2.</b>
<b>3.</b>

<b>What is the expected output? What do you see instead?</b>
Cast/Writer/Director .xml files having &lt;library count=&quot;1&quot;&gt; but instead I see &lt;library count=&quot;0&quot;&gt; which gives DET:Problem with file: null error.

<b>What version/revision of YAMJ are you using? On what operating system?</b>
Yamj r2939

<b>What skin are you using?</b>
Eversion+terraNOVA v3.6

<b>What device are you using YAMJ on? Networked Media Tank, PlayonHD, Other?</b>
PCH A-210

<b>Please provide any additional information below.</b>

Basically this is what happens.

New jukebox scan using moviejukebox.properties.
mjb.categories.minCount=1

this creates cast/writer/director/ *.xml files with &lt;library count=&quot;1&quot;&gt;

if I re-run yamj scan with:
mjb.categories.minCount=2

this creates cast/writer/director/ *.xml files with &lt;library count=&quot;0&quot;&gt;

this gives the error when running Eversion+any skin, within cast (selecting ie. bruce willis) having &lt;library count=&quot;0&quot;&gt;, error being 'DET:Problem with file: null'

This also seems to apply to 'Person' *.xml files.

If needed, I'll zip up all files to replicate this problem.
